# shark_utils/config/trigger_params.yaml

rosout_trigger:
  ros__parameters:
    target_node_name: 'shark1.bt_navigator'
    target_level: 20
    target_substring: 'Goal succeeded'
    run_cmd: 'bash -c "source ~/catkin_ws/install/setup.bash && ros2 run aruco_imu_aligner aruco_imu_aligner"'
    #                                                                                                      
